conf/messages

```diff
@@ -205,6 +204,13 @@ cannotUseService.a = Make a full custom declaration
 cannotUseService.href = https://www.gov.uk/guidance/making-a-full-import-declaration
 
 
+# Custom Agent
+customsDeclares.title = Are you a customs agent?
```

Comment:
should prefix not be customsAgent as that is the name of the View/Controller etc